addons/popochiu/engine/cursor/cursor.gd

```diff
@@ -67,35 +70,43 @@ func show_cursor(anim_name := "normal", ignore_block := false) -> void:
 	
 	if (
 		not anim_name.is_empty()
-		and not $MainCursor.sprite_frames.has_animation(anim_name)
+		and not main_cursor.sprite_frames.has_animation(anim_name)
 	):
-		prints("[Popochiu] Cursor has no animation: %s" % anim_name)
+		PopochiuUtils.print_error("Cursor has no animation: %s" % anim_name)
```

Comment:
Nice one @mapedorr, can you please explain this util function so that I can take note of that and document it for contributors?

Reply:
Sure.

I created a couple of functions to print messages in the output with a special style (using bbcode) so we can use when a message is from our plugin. At the moment, we have: `print_error(msg: String)` which shows the message in red, `print_warning(msg: String)` which shows the message in yellow, and `print_normal(msg: String)` which shows the message in green. All of them use the colors of the palette used by Popochiu: https://lospec.com/palette-list/colodore.

Now that you mentioned, I have to finish the code documentation for the **PopochiuUtils** class.